### ISSUE TITLE
Org-mode style collapsible headings

### ISSUE THREAD

**Issue by galenbwill** [jonpalmisc/bn-notepad#5]

**Comment by jonpalmisc** [comment]
I like this idea. Will investigate sometime after #2.

**Comment by jonpalmisc** [comment]
Going to close this as out-of-scope for now. If there is a demand for it, the issue can be reopened. I don't plan on adding this in the near future, though if you would like to I'd happily accept a PR.
